sentry_sdk/integrations/redis.py

```diff
@@ -15,6 +15,27 @@
 _MULTI_KEY_COMMANDS = frozenset(["del", "touch", "unlink"])
 
 
+def _patch_rediscluster():
+    # type: () -> None
+    try:
+        import rediscluster
+    except ImportError:
+        pass
```

Comment:
Should we do a "return" here perhaps? Otherwise the code below will fail.

Reply:
Ah, you're catching everything when in the parent function. Still, would probably be nice to return, because it doesn't make a lot of sense to continue if the package wasn't imported properly.

Reply:
good catch. that should definitely be a return!